capstone/src/main/java/com/step902020/capstone/EventController.java

```diff
@@ -89,8 +89,8 @@
   }
 
   @GetMapping("get-map-events")
-  public Iterable<Event> getMapEvents(CurrentUser user) {
-    University university = this.individualRepository.findFirstByEmail(user.getEmail()).getUniversity();
```

Comment:
I wonder why this doesn't work.

Reply:
This function works if the user is an individual, but if the user is an organization it fails on the first line because it uses individualRepository.

Reply:
I see. OK!